### ISSUE TITLE
Added dependencies force downgrade of pyarrow

### ISSUE THREAD

**Issue by michaelaye** [conda-forge/zstd-feedstock#26]
Issue:
Previous versions of `zstd` had no dependencies. The recently (for versions >1.3.3) added dependencies force `conda` to downgrade `pyarrow` from 0.13 to 0.12.1.
Is that intended?

```bash
$ conda update zstd                                                                          (py37)
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /Users/klay6683/miniconda3/envs/py37

  added / updated specs:
    - zstd


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    arrow-cpp-0.12.1           |   py37h2ddd891_0         5.5 MB  conda-forge
    libprotobuf-3.6.1          |    hd9629dc_1001         3.8 MB  conda-forge
    pyarrow-0.12.1             |   py37hf9e6f3b_0         2.0 MB  conda-forge
    ------------------------------------------------------------
                                           Total:        11.4 MB

The following packages will be UPDATED:

  zstd                                              1.3.3-1 --> 1.4.0-ha9f0a20_0

The following packages will be DOWNGRADED:

  arrow-cpp                           0.13.0-py37hbfb945d_2 --> 0.12.1-py37h2ddd891_0
  libprotobuf                              3.7.1-hfbae3c0_0 --> 3.6.1-hd9629dc_1001
  pyarrow                             0.13.0-py37hb37e6aa_0 --> 0.12.1-py37hf9e6f3b_0


Proceed ([y]/n)? n
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /Users/klay6683/miniconda3/envs/py37:
#
# Name                    Version                   Build  Channel
advbumpversion            1.2.0                      py_0    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
apipkg                    1.5                        py_0    conda-forge
appdirs                   1.4.3                      py_1    conda-forge
appnope                   0.1.0                 py37_1000    conda-forge
argh                      0.26.2                   pypi_0    pypi
arrow-cpp                 0.13.0           py37hbfb945d_2    conda-forge
asn1crypto                0.24.0                py37_1003    conda-forge
astroid                   2.2.5                    py37_0    conda-forge
astropy                   3.1.2            py37h1de35cc_0    conda-forge
atomicwrites              1.3.0                      py_0    conda-forge
attrs                     19.1.0                     py_0    conda-forge
autopep8                  1.4.4                      py_0    conda-forge
babel                     2.6.0                      py_1    conda-forge
backcall                  0.1.0                      py_0    conda-forge
black                     19.3b0                     py_0    conda-forge
bleach                    3.1.0                      py_0    conda-forge
blosc                     1.16.3               h0a44026_0    conda-forge
bokeh                     1.1.0                    py37_0    conda-forge
boost-cpp                 1.68.0            h6f8c590_1000    conda-forge
brotli                    1.0.7             h0a44026_1000    conda-forge
bzip2                     1.0.6             h1de35cc_1002    conda-forge
ca-certificates           2019.3.9             hecc5488_0    conda-forge
cairo                     1.16.0            h9247486_1000    conda-forge
cartopy                   0.17.0          py37h929c6f0_1004    conda-forge
certifi                   2019.3.9                 py37_0    conda-forge
cffi                      1.12.3           py37hccf1714_0    conda-forge
cftime                    1.0.3.4         py37h917ab60_1000    conda-forge
chardet                   3.0.4                 py37_1003    conda-forge
click                     7.0                        py_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
cligj                     0.5.0                      py_0    conda-forge
cloudpickle               0.8.1                      py_0    conda-forge
colorcet                  2.0.1                      py_0    conda-forge
coverage                  4.5.3            py37h1de35cc_0    conda-forge
cryptography              2.6.1            py37h212c5bf_0    conda-forge
cspice                    66                h1de35cc_1004    conda-forge
curl                      7.64.1               h22ea746_0    conda-forge
cycler                    0.10.0                     py_1    conda-forge
cython                    0.29.7           py37h6de7cb9_0    conda-forge
cytoolz                   0.9.0.1          py37h470a237_1    conda-forge
dask                      1.2.2                      py_0    conda-forge
dask-core                 1.2.2                      py_0    conda-forge
datashader                0.7.0                      py_0    conda-forge
datashape                 0.5.4                      py_1    conda-forge
dbus                      1.13.6               h2f22bb5_0    conda-forge
decorator                 4.4.0                      py_0    conda-forge
defusedxml                0.5.0                      py_1    conda-forge
descartes                 1.1.0                      py_3    conda-forge
distributed               1.28.1                   py37_0    conda-forge
docutils                  0.14                  py37_1001    conda-forge
entrypoints               0.3                   py37_1000    conda-forge
execnet                   1.6.0                      py_0    conda-forge
expat                     2.2.5             h0a44026_1002    conda-forge
fastparquet               0.3.1            py37h01d97ff_0    conda-forge
fiona                     1.8.6            py37h7eb7563_3    conda-forge
flake8                    3.7.7                    py37_0    conda-forge
fontconfig                2.13.1            h1027ab8_1000    conda-forge
freetype                  2.10.0               h24853df_0    conda-forge
freexl                    1.0.5             h1de35cc_1002    conda-forge
gdal                      2.4.1            py37h7eb7563_0    conda-forge
geopandas                 0.5.0                      py_2    conda-forge
geos                      3.7.1             h0a44026_1000    conda-forge
geotiff                   1.4.3             hce09ea4_1000    conda-forge
geoviews                  1.6.2                      py_0    conda-forge
geoviews-core             1.6.2                      py_0    conda-forge
gettext                   0.19.8.1          h46ab8bc_1002    conda-forge
gflags                    2.2.2             h0a44026_1001    conda-forge
giflib                    5.1.7                h01d97ff_1    conda-forge
gitchangelog              3.0.4                    pypi_0    pypi
glib                      2.58.3            h2836805_1001    conda-forge
glog                      0.3.5             h0a44026_1001    conda-forge
graphviz                  2.40.1               ha626266_0    conda-forge
h5py                      2.9.0                    pypi_0    pypi
hdf4                      4.2.13            hf3c6af0_1002    conda-forge
hdf5                      1.10.4          nompi_h0cbb7df_1106    conda-forge
heapdict                  1.0.0                 py37_1000    conda-forge
holoviews                 1.12.2                     py_0    conda-forge
hvplot                    0.4.0                      py_1    conda-forge
icu                       58.2              h0a44026_1000    conda-forge
idna                      2.8                   py37_1000    conda-forge
imageio                   2.5.0                    py37_0    conda-forge
imagesize                 1.1.0                      py_0    conda-forge
importlib-resources       1.0.2                    pypi_0    pypi
ipykernel                 5.1.0           py37h24bf2e0_1002    conda-forge
ipympl                    0.2.1                 py37_1001    conda-forge
ipyparallel               6.2.4                    py37_0    conda-forge
ipython                   7.5.0            py37h24bf2e0_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.4.2                      py_0    conda-forge
isort                     4.3.19                   py37_0    conda-forge
jedi                      0.13.3                   py37_0    conda-forge
jinja2                    2.10.1                     py_0    conda-forge
jpeg                      9c                h1de35cc_1001    conda-forge
json-c                    0.13.1            h1de35cc_1001    conda-forge
jsonschema                3.0.1                    py37_0    conda-forge
jupyter                   1.0.0                      py_2    conda-forge
jupyter_client            5.2.4                      py_3    conda-forge
jupyter_console           6.0.0                      py_0    conda-forge
jupyter_contrib_core      0.3.3                      py_2    conda-forge
jupyter_core              4.4.0                      py_0    conda-forge
jupyter_nbextensions_configurator 0.4.1                    py37_0    conda-forge
jupyterlab                0.35.6                   py37_0    conda-forge
jupyterlab-git            0.5.0                    pypi_0    pypi
jupyterlab-github         0.7.0                    pypi_0    pypi
jupyterlab-latex          0.4.1                    pypi_0    pypi
jupyterlab_server         0.2.0                      py_0    conda-forge
kealib                    1.4.10            hecf890f_1003    conda-forge
kiwisolver                1.0.1           py37h04f5b5a_1002    conda-forge
krb5                      1.16.3            hcfa6398_1001    conda-forge
latexcodec                1.0.7                      py_0    conda-forge
lazy-object-proxy         1.3.1           py37h1de35cc_1000    conda-forge
libblas                   3.8.0                5_openblas    conda-forge
libcblas                  3.8.0                5_openblas    conda-forge
libcurl                   7.64.1               h16faf7d_0    conda-forge
libcxx                    8.0.0                         2    conda-forge
libcxxabi                 8.0.0                         2    conda-forge
libdap4                   3.19.1            hae55d67_1000    conda-forge
libedit                   3.1.20170329      hcfe32e1_1001    conda-forge
libffi                    3.2.1             h6de7cb9_1006    conda-forge
libgdal                   2.4.1                h16bfee5_0    conda-forge
libgfortran               3.0.1                         0    conda-forge
libiconv                  1.15              h01d97ff_1005    conda-forge
libkml                    1.3.0             h71ee1b2_1009    conda-forge
liblapack                 3.8.0                5_openblas    conda-forge
libnetcdf                 4.6.2             h6b88ef6_1001    conda-forge
libpng                    1.6.37               h2573ce8_0    conda-forge
libpq                     11.2                 h29c6f28_0    conda-forge
libprotobuf               3.7.1                hfbae3c0_0    conda-forge
libsodium                 1.0.16            h1de35cc_1001    conda-forge
libspatialindex           1.9.0                h6de7cb9_0    conda-forge
libspatialite             4.3.0a            h0cd9627_1026    conda-forge
libssh2                   1.8.2                hcdc9a53_2    conda-forge
libtiff                   4.0.10            h79f4b77_1001    conda-forge
libxml2                   2.9.9                hd80cff7_0    conda-forge
libxslt                   1.1.32            h33a18ac_1002    conda-forge
line_profiler             2.1.2            py37h1de35cc_0  
livereload                2.6.0                    pypi_0    pypi
llvmlite                  0.28.0           py37hf292337_0    conda-forge
locket                    0.2.0                      py_2    conda-forge
lxml                      4.3.3            py37h08abf6f_0    conda-forge
lz4-c                     1.8.3             h0a44026_1001    conda-forge
mapclassify               2.0.1                      py_0    conda-forge
markupsafe                1.1.1            py37h1de35cc_0    conda-forge
matplotlib                3.0.3                    py37_1    conda-forge
matplotlib-base           3.0.3            py37habd5b94_1    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
mistune                   0.8.4           py37h1de35cc_1000    conda-forge
mock                      2.0.0                 py37_1001    conda-forge
more-itertools            4.3.0                 py37_1000    conda-forge
msgpack-python            0.6.1            py37h04f5b5a_0    conda-forge
multipledispatch          0.6.0                      py_0    conda-forge
munch                     2.3.2                      py_0    conda-forge
nb-black                  1.0.4                    pypi_0    pypi
nbconvert                 5.5.0                      py_0    conda-forge
nbformat                  4.4.0                      py_1    conda-forge
nbresuse                  0.3.1                    pypi_0    pypi
nbsphinx                  0.4.2                      py_0    conda-forge
nbstripout                0.3.5                      py_0    conda-forge
nbtools                   0.2.2.dev5+g740bc06           dev_0    <develop>
nbval                     0.9.1                      py_0    conda-forge
ncurses                   6.1               h0a44026_1002    conda-forge
netcdf4                   1.5.1.2          py37h58b02fa_0    conda-forge
networkx                  2.3                        py_0    conda-forge
nodejs                    11.14.0              h6de7cb9_1    conda-forge
notebook                  5.7.8                    py37_0    conda-forge
numba                     0.43.1           py37h051e8ed_0    conda-forge
numexpr                   2.6.9           py37h1702cab_1000    conda-forge
numpy                     1.16.3           py37hdf140aa_0    conda-forge
olefile                   0.46                       py_0    conda-forge
openblas                  0.3.5             h436c29b_1001    conda-forge
openjpeg                  2.3.1                hc1feee7_0    conda-forge
openssl                   1.1.1b               h01d97ff_2    conda-forge
oset                      0.1.3                      py_1    conda-forge
owslib                    0.17.1                     py_0    conda-forge
p4terrains                0.2.1                     dev_0    <develop>
packaging                 19.0                       py_0    conda-forge
pandas                    0.24.2           py37h0a44026_0    conda-forge
pandoc                    2.7.2                         0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
param                     1.9.0                      py_0    conda-forge
parquet-cpp               1.5.1                         2    conda-forge
parso                     0.4.0                      py_0    conda-forge
partd                     0.3.9                      py_0    conda-forge
pathtools                 0.1.2                    pypi_0    pypi
patsy                     0.5.1                      py_0    conda-forge
pbr                       5.1.3                      py_0    conda-forge
pcre                      8.41              h0a44026_1003    conda-forge
pep8                      1.7.1                      py_0    conda-forge
pexpect                   4.7.0                    py37_0    conda-forge
phantomjs                 2.1.1                         1    conda-forge
pickleshare               0.7.5                 py37_1000    conda-forge
pillow                    6.0.0            py37h7095ceb_0    conda-forge
pip                       19.1                     py37_0    conda-forge
pixman                    0.34.0            h1de35cc_1003    conda-forge
planet4                   0.11.2                    dev_0    <develop>
planetpy                  0.4.6                     dev_0    <develop>
pluggy                    0.9.0                      py_0    conda-forge
poppler                   0.67.0               hb974355_6    conda-forge
poppler-data              0.4.9                         1    conda-forge
port-for                  0.3.1                    pypi_0    pypi
postgresql                11.2                 h16efa30_0    conda-forge
proj4                     5.2.0             h1de35cc_1001    conda-forge
prometheus_client         0.6.0                      py_0    conda-forge
prompt_toolkit            2.0.9                      py_0    conda-forge
psutil                    5.6.2            py37h01d97ff_0    conda-forge
ptyprocess                0.6.0                   py_1001    conda-forge
pvl                       0.3.0                      py_1    conda-forge
py                        1.8.0                      py_0    conda-forge
pyaml                     18.11.0                    py_0    conda-forge
pyarrow                   0.13.0           py37hb37e6aa_0    conda-forge
pybtex                    0.22.2                   py37_0    conda-forge
pybtex-docutils           0.2.1                 py37_1000    conda-forge
pyciss                    0.12.6                    dev_0    <develop>
pycodestyle               2.5.0                      py_0    conda-forge
pycparser                 2.19                     py37_1    conda-forge
pyct                      0.4.6                      py_0    conda-forge
pyct-core                 0.4.6                      py_0    conda-forge
pyepsg                    0.4.0                      py_0    conda-forge
pyflakes                  2.1.1                      py_0    conda-forge
pygments                  2.4.0                      py_0    conda-forge
pykdtree                  1.3.1           py37h917ab60_1002    conda-forge
pylint                    2.3.1                    py37_0    conda-forge
pyopenssl                 19.0.0                   py37_0    conda-forge
pyparsing                 2.4.0                      py_0    conda-forge
pyproj                    1.9.6           py37h9c430a6_1000    conda-forge
pyqt                      5.9.2            py37h2a560b1_0    conda-forge
pyrise                    0.6.0                     dev_0    <develop>
pyrsistent                0.14.11          py37h1de35cc_0    conda-forge
pyshp                     2.1.0                      py_0    conda-forge
pysis                     0.6.0                     dev_0    <develop>
pysocks                   1.6.8                 py37_1002    conda-forge
pyspectral                0.8.7                    pypi_0    pypi
pytables                  3.5.1            py37h689b3ec_1    conda-forge
pytelescope               0.1.0                     dev_0    <develop>
pytest                    4.5.0                    py37_0    conda-forge
pytest-arraydiff          0.3                        py_0    conda-forge
pytest-astropy            0.5.0                      py_0    conda-forge
pytest-cache              1.0                      py37_1  
pytest-cov                2.7.1                      py_0    conda-forge
pytest-doctestplus        0.3.0                      py_0    conda-forge
pytest-openfiles          0.3.1                      py_0    conda-forge
pytest-pep8               1.0.6                      py_1    conda-forge
pytest-remotedata         0.3.1                      py_0    conda-forge
python                    3.7.3                h0d93f26_0    conda-forge
python-dateutil           2.8.0                      py_0    conda-forge
python-geotiepoints       1.1.7                    pypi_0    pypi
python-graphviz           0.10.1                     py_0    conda-forge
python-pptx               0.6.18                     py_0    conda-forge
python-snappy             0.5.4            py37h1e06ddd_0    conda-forge
pytz                      2019.1                     py_0    conda-forge
pyuvis                    0.4                       dev_0    <develop>
pyviz_comms               0.7.2                      py_0    conda-forge
pywavelets                1.0.3            py37h1e5eb4f_1    conda-forge
pyyaml                    5.1              py37h1de35cc_0    conda-forge
pyzmq                     18.0.1           py37h2d07e9b_1    conda-forge
qt                        5.9.7                h93ee506_1    conda-forge
qtconsole                 4.4.4                      py_0    conda-forge
re2                       2019.04.01           h6de7cb9_0    conda-forge
readline                  7.0               hcfe32e1_1001    conda-forge
requests                  2.21.0                py37_1000    conda-forge
requirements-parser       0.2.0                      py_0    conda-forge
rtree                     0.8.3           py37h666c49c_1002    conda-forge
scikit-image              0.15.0           py37h6de7cb9_0    conda-forge
scikit-learn              0.20.3           py37hca23c92_1    conda-forge
scipy                     1.2.1            py37hbd7caa9_1    conda-forge
seaborn                   0.9.0                      py_1    conda-forge
selenium                  3.141.0         py37h1de35cc_1000    conda-forge
send2trash                1.5.0                      py_0    conda-forge
setuptools                41.0.1                   py37_0    conda-forge
shapely                   1.6.4           py37h4b8df73_1004    conda-forge
sip                       4.19.8          py37h0a44026_1000    conda-forge
six                       1.12.0                py37_1000    conda-forge
snappy                    1.1.7             h0a44026_1002    conda-forge
snowballstemmer           1.2.1                      py_1    conda-forge
sortedcontainers          2.1.0                      py_0    conda-forge
sphinx                    2.0.1                      py_0    conda-forge
sphinx-autobuild          0.7.1                    pypi_0    pypi
sphinxcontrib-applehelp   1.0.1                      py_0    conda-forge
sphinxcontrib-bibtex      0.4.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.1                      py_0    conda-forge
sphinxcontrib-htmlhelp    1.0.2                      py_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.2                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.1                      py_0    conda-forge
spicer                    0.2b0                     dev_0    <develop>
spiceypy                  2.2.0                      py_0    conda-forge
sqlite                    3.26.0            h1765d9f_1001    conda-forge
statsmodels               0.9.0           py37h917ab60_1000    conda-forge
tblib                     1.3.2                      py_1    conda-forge
terminado                 0.8.2                    py37_0    conda-forge
testpath                  0.4.2                   py_1001    conda-forge
thrift                    0.11.0          py37h0a44026_1001    conda-forge
thrift-cpp                0.12.0            hdedcb34_1002    conda-forge
tk                        8.6.9             ha441bb4_1001    conda-forge
toml                      0.10.0                     py_0    conda-forge
toolz                     0.9.0                      py_1    conda-forge
tornado                   6.0.2            py37h01d97ff_0    conda-forge
tqdm                      4.32.1                     py_0    conda-forge
traitlets                 4.3.2                 py37_1000    conda-forge
tzcode                    2018g             h1de35cc_1001    conda-forge
urllib3                   1.24.3                   py37_0    conda-forge
watchdog                  0.9.0                    pypi_0    pypi
wcwidth                   0.1.7                      py_1    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.33.4                   py37_0    conda-forge
widgetsnbextension        3.4.2                 py37_1000    conda-forge
wrapt                     1.11.1           py37h1de35cc_0    conda-forge
xarray                    0.12.1                     py_0    conda-forge
xerces-c                  3.2.2             h44e365a_1001    conda-forge
xlrd                      1.2.0                      py_0    conda-forge
xlsxwriter                1.1.8                      py_0    conda-forge
xlwt                      1.3.0                      py_1    conda-forge
xz                        5.2.4             h1de35cc_1001    conda-forge
yaml                      0.1.7             h1de35cc_1001    conda-forge
yapf                      0.27.0                     py_0    conda-forge
zeromq                    4.3.1             h0a44026_1000    conda-forge
zict                      0.1.4                      py_0    conda-forge
zlib                      1.2.11            h1de35cc_1004    conda-forge
zstd                      1.3.3                         1    conda-forge

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : py37
    active env location : /Users/klay6683/miniconda3/envs/py37
            shell level : 1
       user config file : /Users/klay6683/.condarc
 populated config files : /Users/klay6683/.condarc
                          /Users/klay6683/miniconda3/envs/py37/.condarc
          conda version : 4.6.14
    conda-build version : not installed
         python version : 3.7.3.final.0
       base environment : /Users/klay6683/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/klay6683/miniconda3/pkgs
                          /Users/klay6683/.conda/pkgs
       envs directories : /Users/klay6683/miniconda3/envs
                          /Users/klay6683/.conda/envs
               platform : osx-64
             user-agent : conda/4.6.14 requests/2.21.0 CPython/3.7.3 Darwin/18.2.0 OSX/10.14.3
                UID:GID : 273771:2260
             netrc file : None
           offline mode : False


```
</details>

**Comment by xhochy** [comment]
@michaelaye This should be fixed, can you retest?

**Comment by michaelaye** [comment]
Not here (on macOS):

```bash
$ conda update --all                                                                                             (py37)
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /Users/klay6683/miniconda3/envs/py37


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    arrow-cpp-0.12.1           |   py37h2ddd891_0         5.5 MB  conda-forge
    libarchive-3.3.3           |    h5c473cc_1005         1.4 MB  conda-forge
    libprotobuf-3.6.1          |    hd9629dc_1001         3.8 MB  conda-forge
    libtiff-4.0.10             |    hd08fb8f_1003         527 KB  conda-forge
    pyarrow-0.12.1             |   py37hf9e6f3b_0         2.0 MB  conda-forge
    ------------------------------------------------------------
                                           Total:        13.3 MB

The following packages will be UPDATED:

  libarchive                            3.3.3-h46c8594_1004 --> 3.3.3-h5c473cc_1005
  libtiff                              4.0.10-h344961e_1002 --> 4.0.10-hd08fb8f_1003
  parquet-cpp                                       1.5.1-2 --> 1.5.1-4
  zstd                                              1.3.3-1 --> 1.4.0-ha9f0a20_0

The following packages will be DOWNGRADED:

  arrow-cpp                           0.13.0-py37h29c365f_3 --> 0.12.1-py37h2ddd891_0
  libprotobuf                              3.7.1-hfbae3c0_0 --> 3.6.1-hd9629dc_1001
  pyarrow                             0.13.0-py37hb37e6aa_0 --> 0.12.1-py37hf9e6f3b_0


Proceed ([y]/n)?
```

**Comment by xhochy** [comment]
@michaelaye Can you give this another try?

**Comment by michaelaye** [comment]
Conda offers me the exact same build, so why should this be improved? I get the exact same feedback as on June 10.

**Comment by michaelaye** [comment]
and the last commit on this feedstock was on April 30, are you pulling my leg?

**Comment by xhochy** [comment]
No, i've spent a considerable amount of fixing into this. The problem is not fixed by editing this feedstock but by the infrastructure consuming it.

* https://github.com/conda-forge/pyarrow-feedstock/pull/77
* https://github.com/conda-forge/arrow-cpp-feedstock/pull/85
* https://github.com/conda-forge/arrow-cpp-feedstock/pull/83
* https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/232
* https://github.com/regro/cf-scripts/pull/512

This should be fixed by updating `zstd`, `arrow-cpp` and `pyarrow`.

**Comment by michaelaye** [comment]
So you did, thanks! I had to reinstall my conda env to make this work. Guess the previous pinning messed up the dependency tree.

**Comment by michaelaye** [comment]
Alas, it still isn't:

```bash
$ conda list pyarrow(py37)
# packages in environment at /Users/klay6683/miniconda3/envs/py37:
#
# Name                    Version                   Build  Channel
pyarrow                   0.11.1          py37hf9e6f3b_1002    conda-forge
klay6683@macl3886|~/s/dans-gdal-scripts (master|✔)
$ conda update pyarrow(py37)
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /Users/klay6683/miniconda3/envs/py37

  added / updated specs:
    - pyarrow


The following packages will be UPDATED:

  arrow-cpp                           0.11.1-py37h3bd774a_1 --> 0.13.0-py37h29c365f_3
  pyarrow                          0.11.1-py37hf9e6f3b_1002 --> 0.13.0-py37hb37e6aa_0
  thrift-cpp         pkgs/main::thrift-cpp-0.11.0-hd79cdb6~ --> conda-forge::thrift-cpp-0.12.0-hdedcb34_1002

The following packages will be DOWNGRADED:

  libtiff                              4.0.10-hd08fb8f_1003 --> 4.0.10-h344961e_1002
  zstd                                     1.4.0-ha9f0a20_0 --> 1.3.3-1


Proceed ([y]/n)? n
```

Maybe the corner case is really my conda-forge based pinning?

```bash
$ conda config --show-sources                                                         (py37)
==> /Users/klay6683/.condarc <==
pinned_packages: []
report_errors: True
anaconda_upload: True

==> /Users/klay6683/miniconda3/envs/py37/.condarc <==
pinned_packages:
  - conda-forge::opencv
  - conda-forge::gdal
  - conda-forge::numpy
channels:
  - conda-forge
  - defaults
```

**Comment by michaelaye** [comment]
This is actually solved by updating to `conda 4.7.5` that was just released, including my strict pinning to `conda-forge`!

```bash
$ conda list zstd                                                                     (py37)
# packages in environment at /Users/klay6683/miniconda3/envs/py37:
#
# Name                    Version                   Build  Channel
zstd                      1.4.0                ha9f0a20_0    conda-forge
klay6683@macl3886|~/D/s/c/p4catalog (master|✚2)
$ conda list pyarrow                                                                  (py37)
# packages in environment at /Users/klay6683/miniconda3/envs/py37:
#
# Name                    Version                   Build  Channel
pyarrow                   0.13.0           py37he1943e6_2    conda-forge
klay6683@macl3886|~/D/s/c/p4catalog (master|✚2)
$ conda update --all                                                                  (py37)
Collecting package metadata (current_repodata.json): done
Solving environment: done

# All requested packages already installed.
```
